app/views/external_users/claims/case_details/_fields.html.haml

```diff
@@ -20,25 +20,22 @@
       .form-row.js-typeahead
         .form-col{class: error_class?(@error_presenter, :external_user)}
           %a{:id => :external_user}
-          %label{:for => "claim_external_user_id_input"}
+          %label.form-label{:for => "claim_external_user_id_input"}
             = t('.advocate')
           = f.collection_select :external_user_id, @provider.advocates, :id, :name_and_number, { include_blank: ''.html_safe }, {class: 'form-control typeahead'}
           = validation_error_message(@error_presenter, :external_user)
 
-        .form-col
-          = f.adp_text_field :providers_ref, label: t('common.external_user.providers_ref'), errors: @error_presenter
+    .form-row
+      .form-col.forced-width
+        = f.adp_text_field :providers_ref, label: t('common.external_user.providers_ref'), errors: @error_presenter, hint_text: t('common.external_user.providers_ref_hint')
 
-    - else
-      .form-row
-        .form-col
-          = f.adp_text_field :providers_ref, label: t('common.external_user.providers_ref'), errors: @error_presenter
 
```

Comment:
@colinbruce @jsugarman 

This is the only thing of interest really. I removed the `else` as the input is the same for both states.